torch/csrc/jit/mobile/import.cpp

```diff
@@ -303,6 +304,18 @@ void BytecodeDeserializer::parseFunctionSchema(
   }
 }
 
+void BytecodeDeserializer::init_upgrader(mobile::Function* function) {
+  for (auto& mobile_code_data : kUpgraderByteCode) {
+    function->append_function(mobile::Function::registerFunc(
```

Comment:
Does it mean that when loading each model, this function will always be called? We should cache the upgraders like @zhxchen17 suggested in his comment.

Reply:
Oh, it may be in #67730.

Reply:
Oh that's a good point. I update the pr and only operators will be initialized the first time when a model is loaded. 

For `kUpgraderByteCode` (the upgrader function), it's initialized in `upgrader_mobile.h` as a static variable. So I don't think they will be executed twice?

Reply:
Chat offline and update in the latest commit following Zhengxu's suggestion.